builtin/providers/azurerm/resource_arm_subnet_test.go

```diff
@@ -30,6 +31,44 @@ func TestAccAzureRMSubnet_basic(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMSubnet_update(t *testing.T) {
+
+	ri := acctest.RandInt()
+	preConfig := fmt.Sprintf(testAccAzureRMSubnet_basic, ri, ri, ri, ri, ri)
+	postConfig := fmt.Sprintf(testAccAzureRMSubnet_update, ri, ri, ri, ri, ri)
+	subscriptionID := os.Getenv("ARM_SUBSCRIPTION_ID")
```

Comment:
Can someone suggest how to get the subscription ID from testAccProviders ? I've been unable to.